### ISSUE TITLE
Option to mute all notifications

### ISSUE THREAD

**Issue by dvalter** [Tobi823/ffupdater#288]
Currently all FFUpdater notifications are created with default priority and sounds.
These parameters may be changed in notification channel preferences per each channel, but only after this channel was created **with a notification**.

It would be nice to have a preference to change/silence new channels by default so a new browser update or a sudden background error won't annoy users in case these sounds are unwanted.

**Comment by Tobi823** [comment]
I see your problem but I don't want to add new options to FFUpdater because it makes testing and maintaining more difficult.

But I can:
- 1) set the default importance level of new notification channels to IMPORTANCE_LOW (no sound).
- 2) use one notification channel for all app update notifications instead of one channel per app.
- 3) create a notification channel "default notification settings". When a new notification channel is created, it takes the settings from this "default notification settings" setting. But ""default notification settings" will never be used

I prefer 2 - I think that nobody uses this fine-grained control notification. If the update notification for Firefox is too annoying, the update notification for Focus will be annoying, too.

What do you think?
